Packs/Core/Scripts/SearchIndicatorInEvents/SearchIndicatorInEvents.yml

```diff
@@ -0,0 +1,48 @@
+commonfields:
+  id: SearchIndicatorInEvents
+  version: -1
+name: SearchIndicatorInEvents
+script: ''
+type: python
+tags:
+  - Utilities
+comment: Searches for a specific indicator in the tenant's events and logs data and extract the logs which the indicator appears in.
+enabled: true
+args:
+  - name: indicator
+    required: true
+    description: The indicator value (e.g., IP address, domain, hash) to search for in the selected dataset logs.
+  - name: time_frame
+    required: true
+    defaultValue: 7 days
+    description: The time range to search within, specified in days (e.g., "7 days" means searching the past 7 days of data).
```

Comment:
The search timeframe in days (e.g., "7 days" means searching the past 7 days of data).